docs/content/direct/hub-chart.md

```diff
@@ -0,0 +1,229 @@
+A KubeStellar Hub installation compatible with the common setup suitable for Common Setup described in the [examples](./examples.md) could be achieved with the following command:
+
+```shell
+helm upgrade --install ks-hub oci://ghcr.io/kubestellar/kubestellar/hub-chart --version $KUBESTELLAR_VERSION \
+  --set-json='ITSes=[{"name":"its1"}]' \
+  --set-json='WDSes=[{"name":"wds1"}]'
+```
+
+After the initial instllation is completed, there are two main ways to install additional control planes (_e.g._, create a second `wds2` WDS):
+
+1. Upgrade the initial chart. This choice requires to relist the existing control planes, which would otherwise be deleted:
+
+```shell
+helm upgrade --install ks-hub oci://ghcr.io/kubestellar/kubestellar/hub-chart --version $KUBESTELLAR_VERSION \
+  --set-json='ITSes=[{"name":"its1"}]' \
+  --set-json='WDSes=[{"name":"wds1"},{"name":"wds2"}]'
+```
+
+2. Install a new chart with a different name. This choice does not requires to relist the existing control planes, but requies to disable the reinstallation of KubeFlex and PCHs:
+
+```shell
+helm upgrade --install add-wds2 oci://ghcr.io/kubestellar/kubestellar/hub-chart --version $KUBESTELLAR_VERSION \
+  --set='kubeflex-operator.install=false,InstallPCHs=false' \
+  --set-json='WDSes=[{name":"wds2"}]'
+```
+
+## Obtaining the Control Planes kubeconfigs
+
+The following code can be used to wait for the Ready state of all the KubeFlex Control Planes in the hub cluster:
+
+```shell
+echo "Waiting for all Control Planes to be ready..."
+for cpname in `kubectl get controlplane -o name`; do
+  cpname=${cpname##*/}
+  while [[ $(kubectl get cp $cpname -o 'jsonpath={..status.conditions[?(@.type=="Ready")].status}') != "True" ]]; do
+    echo "Waiting for Control Plane $cpname..."
+    sleep 5
+  done
+  echo $cpname
+done
+```
+
+The following code can be used to obtain the kubeconfig of all the KubeFlex Control Planes in the hub cluster:
+
+```shell
+echo "Getting the kubeconfig of all Control Planes..."
+for cpname in `kubectl get controlplane -o name`; do
+  cpname=${cpname##*/}
+  echo "Getting the kubeconfig of Control Planes $cpname ==> /tmp/kubeconfig-$cpname..."
+  if [[ "$(kubectl get controlplane $cpname -o=jsonpath='{.spec.type}')" == "host" ]] ; then
+    kubectl config view --minify --flatten > "kubeconfig-$cpname"
+  else
+    kubectl get secret $(kubectl get controlplane $cpname -o=jsonpath='{.status.secretRef.name}') \
+      -n $(kubectl get controlplane $cpname -o=jsonpath='{.status.secretRef.namespace}') \
+      -o=jsonpath="{.data.$(kubectl get controlplane $cpname -o=jsonpath='{.status.secretRef.key}')}" \
+      | base64 -d > "kubeconfig-$cpname"
+  fi
+  kubectl --kubeconfig "kubeconfig-$cpname" config rename-context $(kubectl --kubeconfig "kubeconfig-$cpname" config current-context) $cpname 2> /dev/null
+done
+```
+
+The code above will saves the kubeconfig of a control plane `$cpname` to a corresponding file `kubeconfig-$cpname` in the local folder.

```

Comment:
You might want to mention, here and in the other copy of this text, the naming of the current context too.

Reply:
ok done both